examples/MuscleOxygenMonitor/MuscleOxygenMonitor.ino

```diff
@@ -0,0 +1,60 @@
+/**********************************************
```

Comment:
there was no merge from the MOxy branch, accidental merge into develop?

Reply:
Yes, accidental. See above.